src/os/bluestore/BlueStore_objects.h

```diff
@@ -65,6 +67,60 @@
 #include "common/zipkin_trace.h"
 #endif
 
+class FixedPoolMemoryResource : public std::pmr::memory_resource {
```

Comment:
Can you detail (pref. as code comments) why is this memory resource needed? (i.e. - why you could not have used the synchronized_pool_resource)?
Also - as it is probably needed - please consider placing it in 'src/common'.